### PR TITLE
chore: update prometheus input readme to match config options

### PR DESCRIPTION
**Pull request by russorat**
### Required for all PRs:

- [ ] Updated associated README.md.
- [ ] Wrote appropriate unit tests.

resolves #

### CODE REVIEW

File: plugins/inputs/prometheus/README.md [hunk 1]
```diff
@@ -19,30 +19,37 @@ in Prometheus format.
...
   ## Only for node scrape scope: node IP of the node that telegraf is running on.
   ## Either this config or the environment variable NODE_IP must be set.
   # node_ip = "10.180.1.1"
-  ## Only for node scrape scope: interval in seconds for how often to get updated pod list for scraping
-  ## Default is 60 seconds.
-  # pod_scrape_interval = 60
+	
+  ## Only for node scrape scope: interval in seconds for how often to get updated pod list for scraping.
+	## Default is 60 seconds.
+	# pod_scrape_interval = 60
```
Comment (Hipska) [inline plugins/inputs/prometheus/README.md:51]:
```suggestion
  ## Default is 60 seconds.
  # pod_scrape_interval = 60
```

**Review by Hipska — CHANGES_REQUESTED** [review]

File: plugins/inputs/prometheus/README.md [hunk 2]
```diff
@@ -19,30 +19,37 @@ in Prometheus format.
   ##   example: metric_version = 1; 
   ##            metric_version = 2; recommended version
   # metric_version = 1
-
+  
+  ## Url tag name (tag containing scrapped url. optional, default is "url")
+  # url_tag = "scrapeUrl"
```
Comment (Hipska) [inline plugins/inputs/prometheus/README.md:24]: The value should represent the default
```suggestion
  # url_tag = "url"
```

**Review by Hipska — CHANGES_REQUESTED** [review]

File: plugins/inputs/prometheus/README.md (hunk 2 above)
Comment (russorat) [reply plugins/inputs/prometheus/README.md:24]: if we change it here, we need to change it here as well: https://github.com/influxdata/telegraf/blob/master/plugins/inputs/prometheus/prometheus.go#L94

Comment (Hipska) [reply plugins/inputs/prometheus/README.md:24]: Correct!

**Review by Hipska — APPROVED** [review]

**Pull request merged**